src/api/goods.ts

```diff
@@ -8,12 +8,28 @@ export function getAll(): Promise<Good[]> {
     .then(response => response.json());
 }
 
-export const get5First = () => {
-  return getAll()
-    .then(goods => goods); // sort and get the first 5
+export const get5First = async () => {
+  const loadedGoods = await getAll()
+    .then(goods => goods);
```

Comment:
don't mix `async/await/try/catch` with `then/catch` in one project

Reply:
How can I do sort without use async? "Property 'sort' does not exist on type 'Promise<Good[]>'.ts(2339)
goods.ts(15, 15): Did you forget to use 'await'?"